smriprep/cli/run.py

```diff
@@ -345,6 +348,9 @@ def build_workflow(opts, retval):
     subject_list = collect_participants(
         layout, participant_label=opts.participant_label)
 
+    bids_filters_file = opts.bids_filters.resolve()
+    bids_filters = json.load(open(bids_filters_file)) if bids_filters_file else None
```

Comment:
Using `Path.read_text()` will avoid leaving the filehandle open and generating a warning.

```suggestion
    bids_filters = json.load(opts.bids_filters.read_text()) if opts.bids_filters else None
```